phpunit/phpunit/4.7/phpunit.xml.dist

```diff
@@ -24,4 +24,18 @@
             <directory suffix=".php">src</directory>
         </whitelist>
     </filter>
+
+    <!-- Run `composer require symfony/phpunit-bridge` before enabling this extension -->
+    <!--
+    <listeners>
+        <listener class="Symfony\Bridge\PhpUnit\SymfonyTestsListener" />
+    </listeners>
+    -->
+
+    <!-- Run `composer require symfony/panther` before enabling this extension -->
+    <!--
+    <extensions>
+        <extension class="Symfony\Component\Panther\ServerExtension" />
+    </extensions>
+    -->
```

Comment:
This is just to make this file consistent with the `phpunit.xml.dist` file from the `phpunit-bridge` recipe, in case someone installs `phpunit` directly (and not via the pack, where you will get the `phpunit.xml.dist` file from `phpunit-bridge`)

Reply:
A nice addition would be to have a PHPUnit configurator in Flex.